components/x11/libXfont/mapfile-vers

```diff
@@ -183,13 +183,9 @@ SUNWprivate {
 	pcfWriteFont;
 	pcfError;
 
-	# "src/bitmap/snfstr.h" - needed by xfs
-	SnfSetFormat;
```

Comment:
@alp There are several CVEs for 1.4.9: CVE-2014-0209, CVE-2014-0210, CVE-2014-0211